### ISSUE TITLE
Scrolling is broken.

### ISSUE THREAD

**Issue by SimonSapin** [bwinton/TabCenter#820]
When there’s enough tabs that they don’t all fit, TabCenter uses a scrollbar. Using the mouse "wheel" while hovering tabs normally scrolls the list of tabs. Since a recent update, nothing happens when I do that.

**Comment by ericawright** [comment]
@SimonSapin Have you restarted firefox since the update?
This was a bug which we saw in testing but should be fixed. If it remains after a restart can you let me know what OS you're on and if you have the 'tabs on top' preference set?

**Comment by SimonSapin** [comment]
Yes, restarting restores scrolling, but reveals another bug: scrolling sometimes switches to the tab that happens to be under the cursor, as if I had clicked on it. It’s kinda disruptive.

**Comment by ericawright** [comment]
okay thank you. I'll look into that.
